packages/custom-field/src/vaadin-custom-field.d.ts

```diff
@@ -100,59 +96,6 @@ export interface CustomFieldEventMap extends HTMLElementEventMap, CustomFieldCus
  * @fires {CustomEvent} validated - Fired whenever the field is validated.
  */
 declare class CustomField extends FieldMixin(FocusMixin(KeyboardMixin(ThemableMixin(ElementMixin(HTMLElement))))) {
```

Comment:
`CustomFieldMixin` needs to be applied here. 

Let's also move `FieldMixin`, `FocusMixin` and `KeyboardMixin` to the mixin definition.

Reply:
Done.